java/ql/src/Security/CWE/CWE-295/AndroidMissingCertificatePinning.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Android Missing Certificate Pinning
+ * @description Network communication should use certificate pinning.
```

Comment:
The description is ok but doesn't follow the preferred syntax of "Syntax X causes behavior Y." 
However this is a minor observation. Feel free to ignore it if you don't agree 😅

Reply:
```suggestion
 * @description Network connections without certificate pinning may allow attackers to eavesdrop communications.
```